docs/architecture/adr-050-sign-mode-textual-annex1.md

```diff
@@ -203,7 +203,16 @@ Object: /cosmos.gov.v1.Vote
 
 ### `google.protobuf.Timestamp`
 
-Rendered as either ISO8601 (`2021-01-01T12:00:00Z`).
+Rendered using [RFC 3339](https://www.rfc-editor.org/rfc/rfc3339) (a
+simplification of ISO 8601), which is the current recommendation for portable
+time values. The rendering always uses "Z" (UTC) as the timezone. It uses only
+the necessary fractional digits of a second, omitting the fractional part
+entirely if the timestamp has no fractional seconds.
```

Comment:
In https://github.com/cosmos/cosmjs/pull/1147/files#diff-1aee49f4f44f595fc5e4c8a2f0e15fce0b30fab5ea6e9e8f1b2894a917b19fa5 I always used 9 digits.

I'm happy for that version. But please note this removed the ability for lexicographic sorting of timestamps.

Reply:
I suggest that long unbroken strings of numbers hinder legibility (hence the reason for the int renderer). I would advocate for a position separator in the fractional part - but I expect that fractional seconds will be rare in practice, and even then probably limited to milliseconds. That's why I'm using as few digits as possible.

Perhaps we should consider truncating zero seconds too? But I think that would break RFC 3339.

Since the rendering is going to a ledger rather than a data file, I think the legibility is more important than the sortability.

Reply:
+1 for truncating unnecessary digits